services/src/test/scala/cromwell/services/healthmonitor/HealthMonitorServiceActorSpec.scala

```diff
@@ -1,160 +1,210 @@
 package cromwell.services.healthmonitor
 
-import akka.actor.Props
+import akka.actor.{ActorRef, Props}
 import akka.pattern.ask
 import akka.util.Timeout
-import com.typesafe.config.ConfigFactory
+import com.typesafe.config.{Config, ConfigFactory}
 import cromwell.core.TestKitSuite
-import cromwell.services.healthmonitor.HealthMonitorServiceActor.{GetCurrentStatus, MonitoredSubsystem, StatusCheckResponse, SubsystemStatus, _}
-import cromwell.services.healthmonitor.HealthMonitorServiceActorSpec.{MockHealthMonitorServiceActor, _}
-import org.scalatest.AsyncFlatSpecLike
+import cromwell.services.healthmonitor.HealthMonitorServiceActor._
+import cromwell.services.healthmonitor.HealthMonitorServiceActorSpec._
+import org.scalatest.{Assertion, FlatSpecLike}
+import org.scalatest.concurrent.Eventually
 
-import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
 import scala.language.postfixOps
 
-class HealthMonitorServiceActorSpec extends TestKitSuite with AsyncFlatSpecLike {
+class HealthMonitorServiceActorSpec extends TestKitSuite with FlatSpecLike with Eventually {
   implicit val timeout = Timeout(5.seconds)
 
+  override val patienceConfig = PatienceConfig(timeout = scaled(20 seconds), interval = scaled(1 second))
+  implicit val patience = patienceConfig
+
+  private def eventualStatus(actorRef: ActorRef, ok: Boolean, systems: (MonitoredSubsystem, SubsystemStatus)*): Assertion = {
+    case class NameAndStatus(name: String, status: SubsystemStatus)
+
+    eventually {
+      val resp = Await.result(actorRef.ask(GetCurrentStatus).mapTo[StatusCheckResponse], Duration.Inf)
```

Comment:
ToL: I'm thinking of reducing (eliminating?) the usage hardcoded `Duration.Inf` except when necessary. Perhaps reuse the variable `implicit val timeout` defined above, since the `eventually` is using that anyway.